### PR TITLE
Feature/add update delete

### PR DESCRIPTION
**Pull request by aromal-qburst**
## Changes

+ Add employee details
+ Edit employee details
+ Delete employee details
+ Basic validation of employee details
+ Better UX

### CODE REVIEW

File: js/main.js [hunk 1]
```diff
@@ -125,37 +127,94 @@ const addUpdateModal = function (isAdd, empId) { // Handle modal on update or ad
...
 
+    skillDropdown.onclick = (event) => {
+        if (event.target.dataset.skillid.startsWith('id-')) {
+            const skillInputTag = document.getElementById('emp-skill');
+            const skillName = event.target.innerText;
+            let skillInput = document.getElementById('emp-skill').value.split(', ');
+
+            skillInput.push(skillName);
+
+            if (skillInput[0] == '') {
```
Comment (SabirAmeen) [inline js/main.js]: @aromal-qburst If you are trying to remove whitespace, use trim instead

File: js/main.js [hunk 2]
```diff
@@ -125,37 +127,94 @@ const addUpdateModal = function (isAdd, empId) { // Handle modal on update or ad
...
+        fillUpdateClearForm(true);
+    }
 
-        if (!isAdd) {
-            fillUpdateClearForm(true);
+    const displayErrorMessage = (isClear, validationErrorStatus) => {
+        const displayErrorSection = document.querySelectorAll('.validation-error');
+        displayErrorSection.forEach((val, index) => {
+            if (!isClear && validationErrorStatus[index] == 1) {
+                val.innerHTML = `<span>!Invalid Input!</span>`;
```
Comment (SabirAmeen) [inline js/main.js]: @aromal-qburst Just "invalid input" will be enough

File: css/style.css [hunk 3]
```diff
@@ -992,28 +1034,30 @@ textarea {
...
   #modal-heading {
-    font-size: 28px;
+    font-size: 25px;
+    margin-top: 30px;
   }
 
-  #modal-form .form-item-spacing {
-    flex-direction: column;
+  .position-heading-close img {
+    top: -35px;
```
Comment (SabirAmeen) [inline css/style.css]: @aromal-qburst Position the close with respect to the modal to avoid negative px values

Comment (aromal-qburst) [reply css/style.css]: Done

File: js/main.js (hunk 2 above)
Comment (aromal-qburst) [reply js/main.js]: Done

File: js/main.js (hunk 1 above)
Comment (aromal-qburst) [reply js/main.js]: Done

**Pull request merged**